.github/workflows/qe-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Push qe images for new tag
+on:
+  push:
+    tags:
+      - '*'
+env:
+  IMAGE_REGISTRY: quay.io
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        go:
+          - '1.21'
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Log in to Quay.io
+        uses: redhat-actions/podman-login@v1
+        with:
+          username: ${{ secrets.QUAY_IO_USERNAME }}
+          password: ${{ secrets.QUAY_IO_PASSWORD }}
+          registry: ${{ env.IMAGE_REGISTRY }}
+      - name: create script of building and push images
+        run: |
+          cat <<EOF > build-qe-images.sh
+          OS=linux ARCH=amd64 CRC_INTEGRATION_IMG_VERSION=v${1} make containerized_integration
+          podman push quay.io/crcont/crc-integration:v${1}-linux-amd64
+          OS=windows ARCH=amd64 CRC_INTEGRATION_IMG_VERSION=v${1} make containerized_integration
+          podman push quay.io/crcont/crc-integration:v${1}-windows-amd64
+          OS=darwin ARCH=amd64 CRC_INTEGRATION_IMG_VERSION=v${1} make containerized_integration
+          podman push quay.io/crcont/crc-integration:v${1}-darwin-amd64
+          OS=darwin ARCH=arm64 CRC_INTEGRATION_IMG_VERSION=v${1} make containerized_integration
+          podman push quay.io/crcont/crc-integration:v${1}-darwin-arm64
+          OS=linux ARCH=amd64 CRC_E2E_IMG_VERSION=v${1} make containerized_e2e
+          podman push quay.io/crcont/crc-e2e:v${1}-linux-amd64
+          OS=windows ARCH=amd64 CRC_E2E_IMG_VERSION=v${1} make containerized_e2e
+          podman push quay.io/crcont/crc-e2e:v${1}-windows-amd64
+          OS=darwin ARCH=amd64 CRC_E2E_IMG_VERSION=v${1} make containerized_e2e
+          podman push quay.io/crcont/crc-e2e:v${1}-darwin-amd64
+          OS=darwin ARCH=arm64 CRC_E2E_IMG_VERSION=v${1} make containerized_e2e
+          podman push quay.io/crcont/crc-e2e:v${1}-darwin-arm64
+          EOF
+      - name: run script to build and push images
+        env:
+          TAG_CONTEXT: ${{ toJson(github.ref_name) }}
```

Comment:
Is `toJson()` func required here?

Reply:
yes, `toJson` is not needed. I have removed it.